maps/empty.go

```diff
@@ -19,16 +19,20 @@ func (m EmptyMap) Meta() Metadata {
 		Name:        "Empty",
 		Description: "Default snake placement with no food",
 		Author:      "Battlesnake",
-		Version:     1,
+		Version:     2,
 		MinPlayers:  1,
-		MaxPlayers:  8,
-		BoardSizes:  AnySize(),
+		MaxPlayers:  16,
+		BoardSizes:  OddSizes(rules.BoardSizeSmall, rules.BoardSizeXLarge),
 	}
 }
 
 func (m EmptyMap) SetupBoard(initialBoardState *rules.BoardState, settings rules.Settings, editor Editor) error {
 	rand := settings.GetRand(0)
 
+	if len(initialBoardState.Snakes) > int(m.Meta().MaxPlayers) {
```

Comment:
There were tests that expected this error for this map type, but the placement algorithm can technically handle more than this. So I thought it didn't hurt to have the map just validate it hasn't exceeded its max players.